packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue

```diff
@@ -75,12 +75,33 @@ export default {
         isOpen: {
             type: Boolean,
             default: false
+        },
+        isBelowMid: {
```

Comment:
Maybe not one for this PR, but for a future EE ticket, would be to make this property part of the state, so that you don't need to pass it down through props. Would likely be a little tidier that way potentially (but not needed as part of this PR)